test/etools-datepicker_test.html

```diff
@@ -20,10 +21,27 @@
 
     <script>
       suite('etools-datepicker', function() {
+        var basic
+        setup(function() {
+          basic = fixture('basic')
+        });
         test('instantiating the element works', function() {
-          var element = fixture('basic');
-          assert.equal(element.is, 'etools-datepicker');
+          assert.equal(basic.is, 'etools-datepicker');
+        });
+        test('tapping datepicker icon toggles it open', function() {
+          expect(dialog.attributes[4].name).to.equal('aria-hidden')
```

Comment:
Using `dialog.attributes[4].name` is not ideal and I do not see `aria-hidden` on the element (this attribute is on `paper-dialog` element).
You could try and use `opened` property instead then check `aria-hidden`(see `hasAttribute` method) on `paper-dialog` inside `basic`. Also you can check in the overlay is opened  querying for `iron-overlay-backdrop` in body.

Reply:
good call. i will change it to check for dialog.opened